src/main.sh

```diff
@@ -20,36 +20,39 @@ main() {
 
   # Check if there is a task link in the PR
   local -r pr_body=$(github::get_pr_body)
-  local -r task_id=$(teamwork::get_task_id_from_body "$pr_body" )
+  local -r task_ids_str=$(teamwork::get_task_id_from_body "$pr_body" )
 
-  if [ "$task_id" == "" ]; then
+  if [ "$task_ids_str" == "" ]; then
     log::message "Task not found"
     exit 0
   fi
 
-  log::message "Task found with the id: $task_id"
-
-  export TEAMWORK_TASK_ID=$task_id
-
-  local -r event=$(github::get_event_name)
-  local -r action=$(github::get_action)
-
-  log::message "Event: $event - Action: $action"
-
-  if [ "$event" == "pull_request" ] && [ "$action" == "opened" ]; then
-    teamwork::pull_request_opened
-  elif [ "$event" == "pull_request" ] && [ "$action" == "closed" ]; then
-    teamwork::pull_request_closed
-  elif [ "$event" == "pull_request_review" ] && [ "$action" == "submitted" ]; then
-    teamwork::pull_request_review_submitted
-  elif [ "$event" == "pull_request_review" ] && [ "$action" == "dismissed" ]; then
-    teamwork::pull_request_review_dismissed
-  elif [ "$event" == "pull_request_review_comment" ] && [ "$action" == "deleted" ]; then
-    teamwork::pull_request_review_comment_deleted
-  else
-    log::message "Operation not allowed"
-    exit 0
-  fi
+  IFS=',' read -r -a task_ids <<< "$task_ids_str"
+  for task_id in "${task_ids[@]}"; do
+    log::message "Task found with the id: $task_id"
+
+    export TEAMWORK_TASK_ID=$task_id
+
+    local -r event=$(github::get_event_name)
+    local -r action=$(github::get_action)
+
+    log::message "Event: $event - Action: $action"
+
+    if [ "$event" == "pull_request" ] && [ "$action" == "opened" ]; then
+      teamwork::pull_request_opened
+    elif [ "$event" == "pull_request" ] && [ "$action" == "closed" ]; then
+      teamwork::pull_request_closed
+    elif [ "$event" == "pull_request_review" ] && [ "$action" == "submitted" ]; then
+      teamwork::pull_request_review_submitted
+    elif [ "$event" == "pull_request_review" ] && [ "$action" == "dismissed" ]; then
+      teamwork::pull_request_review_dismissed
+    elif [ "$event" == "pull_request_review_comment" ] && [ "$action" == "deleted" ]; then
+      teamwork::pull_request_review_comment_deleted
+    else
+      log::message "Operation not allowed"
+      exit 0
+    fi
```

Comment:
We could move this to a function that would receive the `$event` and `$task_id`.

Reply:
Maybe we can just move outside of the loop the lines 36-39.

I did today another PR removing some unused code, so the code inside the "for" won't be so long.